lib/modules/dosomething/dosomething_signup/dosomething_signup.signup_data_form.inc

```diff
@@ -507,6 +507,10 @@ function dosomething_signup_user_signup_data_form_submit($form, &$form_state) {
   }
   // Update signup record.
   dosomething_signup_update_signup_data($values, $config);
+  // If this is a competition sign up, send it to gladiator.
+  if ($config['competition_signup']) {
+    dosomething_signup_send_user_to_gladiator($user, $values);
```

Comment:
I think we will probably want some other feature flag here, but again, that can come later